### ISSUE TITLE
Is this plugin working

### ISSUE THREAD

**Issue by cubewebsolutions** [envato/wp-envato-market#59]
I noticed recently the older Envato plugin was not allowing me to update themes. So I sent searching and found that I have to use this one now. So I've installed it and connected with my token successfully.

Now when I go to the themes, how am I supposed to update them like I used to be able to on the old plugin? 

For example, have a look at this one:

![image](https://cloud.githubusercontent.com/assets/8720991/17456835/ee9bc1c0-5c27-11e6-8a5c-3046f67a17e2.png)

There is a new version waiting, but no update button?

Am I missing something?

**Comment by dtbaker** [comment]
There have been some recent API changes. Hey please let us know if this is still an issue and we can look into it further.

**Comment by Evelyn7** [comment]
Hi+
I downloaded the zip file from your website on 02/26/18, then removed the old plugin and installed the new zip file via the back-end Plugins > add new > upload. I then activated - and it still did not work. It did show an "update" button/link, however, when I clicked that nothing happened.

WP v 4.9.4
Envato Plugin v. 1.0.0-RC2

So, is the zip file on your website the "new" (fixed) version of the plugin? If so, why is it not working as expected? If not, where do I find the correct version?
Thanks!
